Magento2/ruleset.xml

```diff
@@ -189,6 +189,10 @@
         <exclude-pattern>*Test.php</exclude-pattern>
         <exclude-pattern>*/tests/*</exclude-pattern>
     </rule>
+    <rule ref="Magento2.Namespaces.ImportsFromTestNamespace">
+        <severity>8</severity>
+        <type>warning</type>
```

Comment:
Please exclude tests, like this:
```suggestion
        <type>warning</type>
        <exclude-pattern>*/_files/*</exclude-pattern>
        <exclude-pattern>*/Fixtures/*</exclude-pattern>
        <exclude-pattern>*/Test/*</exclude-pattern>
        <exclude-pattern>*Test.php</exclude-pattern>
        <exclude-pattern>*/tests/*</exclude-pattern>
```

Reply:
Actually it's so minor, I added it myself

Reply:
Thanks @ihor-sviziev